src/main/java/useCases/Timer/TimerInteractor.java

```diff
@@ -10,10 +10,6 @@
  */
 public class TimerInteractor {
```

Comment:
This should implement an Input Boundary

Reply:
Yes, I might add that. But Derek today also told me it might be okay to not have an implemented interface because the only input data I really need is a Duration object, but I am still thinking about this design choice. Strongly considering creating an interface.